GoogleForCreators/web-stories-wp
# Carousel: Enable page thumbnail caching by default

## Context



## Summary
Sets the feature flag for caching page thumbnails to `true` now that issues for Safari have been fixed within https://github.com/google/web-stories-wp/pull/9168.


## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Open the editor in Safari
2. Add a few pages with different images etc.
3. Verify that when navigating between the pages, the page thumbnails display the page content correctly and not just empty / gray as previously.
Note: some shapes display incorrectly for Safari, please ignore that, see #9173 


## Checklist




---



Fixes #9263
